web/packages/teleport/src/TopBar/TopBar.tsx

```diff
@@ -150,6 +152,10 @@ export function TopBar({ hidePopup = false }: TopBarProps) {
       })
     );
 
+  function handleBack() {
+    history.goBack();
```

Comment:
If I visit a feature with `hideNavigation = true` and it's the first URL that I visit, this is going to redirect me to `about:blank`.

Initially I was about to suggest that maybe this should redirect to the top feature from a respective category. But for now maybe a simpler solution would be to return to the main Web UI page if there's nothing to go back to?

Reply:
I changed this to push the user to the first feature in the active category